source/features/linkify-labels-on-dashboard.tsx

```diff
@@ -8,9 +8,14 @@ import {wrap} from '../helpers/dom-utils';
 import features from '.';
 
 function init(): void {
-	observe('.js-recent-activity-container :not(a) > .IssueLabel', {
+	const labelClass = [
+		'.js-recent-activity-container :not(a) > .IssueLabel:not(.rgh-linkfied-label)', // Recent activity
+		'.js-all-activity-header + div :not(a) > .IssueLabel:not(.rgh-linkfied-label)' // Newsfeed
+	].join();
+	observe(labelClass, {
 		add(label) {
-			const activity = label.closest('li')!;
+			label.classList.add('rgh-linkfied-label');
+			const activity = label.closest('li')! ?? label.parentElement; // Newsfeed
```

Comment:
Can this be merged? Random example:
```suggestion
			const activity = label.closest('li, div')!;
```

Also, never use `! ??`, because `??` already handles and null value.

Reply:
Need to test it (give me a few min working on anther PR) but I _think_ there is a `div` there that will get caught.

Reply:
> Can this be merged

No. There is a `div` in the non-newsfeed there that will get caught.

Reply:
That was _random example_ to ask you to find a common selector

Reply:
[`63d9957` (#3625)](https://github.com/sindresorhus/refined-github/pull/3625/commits/63d99571f38f527cf05cbf04b5c652684280d9d1) How's this?